projects/RabbitMQ.Client/client/impl/SessionBase.cs

```diff
@@ -32,6 +32,7 @@
 using System;
 using System.Collections.Generic;
 using System.Threading;
+
```

Comment:
Is this intentional?

Reply:
VS autoformatter added this. It's set to split usings between base-namespaces. I can remove if wanted.